workflow/executor/agent.go

```diff
@@ -260,18 +260,35 @@ func (ae *AgentExecutor) executeHTTPTemplate(ctx context.Context, tmpl wfv1.Temp
 			message = fmt.Sprintf("received non-2xx response code: %d", response.StatusCode)
 		}
 	} else {
-		evalScope := map[string]interface{}{
-			"request": map[string]interface{}{
-				"method":  tmpl.HTTP.Method,
-				"url":     tmpl.HTTP.URL,
-				"body":    tmpl.HTTP.Body,
-				"headers": tmpl.HTTP.Headers.ToHeader(),
-			},
-			"response": map[string]interface{}{
-				"statusCode": response.StatusCode,
-				"body":       string(bodyBytes),
-				"headers":    response.Header,
-			},
+		var evalScope map[string]interface{}
+		if tmpl.HTTP.BodyFrom != nil {
```

Comment:
@simster7 has good point. Can you add `GetBody` function in `HTTP` which will return body for request?

http_type.go
```go
func (h *HTTP) GetBody() (string, error){
	if h.Body != "" {
		return h.Body, nil
	}
	if h.BodyFrom != nil && h.BodyFrom.Bytes != nil {
		return string (h.BodyFrom.Bytes), nil
	}
	return "", fmt.Errorf("HTTP body is not found")
}
```